File: waterloo-bank/app/src/main/java/uk/co/waterloobank/AccountActivity.java
package uk.co.waterloobank;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class AccountActivity extends Activity {

    private TextView tokenTextView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_account);

        setTitle("Your Account");

        String token = getIntent().getStringExtra("token");

        tokenTextView = (TextView) findViewById(R.id.tokenTextView);
        tokenTextView.setText("Your token is: " + token);
    }
}

File: waterloo-bank/app/src/main/java/uk/co/waterloobank/AccountActivity.kt
package uk.co.waterloobank

import android.app.Activity
import android.os.Bundle
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_account.*

class AccountActivity: Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        
        setContentView(R.layout.activity_account)

        title = "Your Account"

        tokenTextView.text = "Your token is: ${intent.getStringExtra("token")}";
    }
}
